lib/Database.js

```diff
@@ -7,11 +7,11 @@ function Database(){}
 Database.prototype.open = function( path, options ){
 
   // set up a safe environment for running tests.
-  // note: usually in-memory databases using the same
-  // path would share the same database reference.
+  // note: each instance of an in-memory database that's created is unique
+  // this means that we can ignore the 'path' parameter and use ':memory:'
+  // https://www.sqlite.org/inmemorydb.html
   if( options && true === options.test ){
-    options.memory = true;
-    path += Math.random().toString(36).substr(2);
+    path = ':memory:';
```

Comment:
@missinglink, one of the changes in `better-sqlite-7` is dropping the `memory: true` flag in favor of the ':memory:' path option. Looking at the SQLite docs it appears that this will work fine for us, creating multiple in-memory databases if we initialize several `Database` class instances in our tests. I've updated the comment here to reflect that but let me know if it's poorly worded or incorrect in any way.